app/models/rummager_search.rb

```diff
@@ -0,0 +1,25 @@
+class RummagerSearch
+  PAGE_SIZE_TO_GET_EVERYTHING = 1000
+
+  def initialize(content_id)
+    @content_id = content_id
```

Comment:
You can set an `attr_reader` for `content_id` which is the equivalent of wrapping `@content_id` in a getter method.